docs/src/development/transfer-dedicated.md

```diff
@@ -51,48 +61,81 @@ Which should give a JSON output containing something like this:
 ]
 ```
 
-The part you want is the user, password, and "path", which means the DB name.
-Ignore the rest.
+From which you need the following values:
+
+- `username`
+- `password` (not needed if empty)
+- `path`: the database name (`<DATABASE_NAME>` in the example command below)
 
-Now, run the following command on your local computer:
+Then, for the environment you want to access, you need to get the [SSH username and host](#get-the-ssh-username-and-host).
+
+Now, adapt and run the following command on your local computer:
 
 ```bash
-ssh <USERNAME>@<CLUSTER_NAME>.ent.platform.sh 'mysqldump --single-transaction -u <user> -p<pass> -h localhost <dbname> | gzip' > database.gz
+ssh <SSH_USERNAME>@<SSH_HOST> 'mysqldump --single-transaction -u <USERNAME> -p <PASSWORD> -h <DATABASE_NAME> | gzip' > database.gz
 ```
 
-That runs a `mysqldump` command on the server, compress it using `gzip`,
-and stream the output to a file named `database.gz` on your local computer.
+The adapted version of the command could look similar to:
+
+{{< codetabs >}}
 
-(If you'd prefer, `bzip2` and `xz` are also available.)
+---
+title=Without a database password
+file=none
+highlight=bash
+---
 
-## Synchronizing files from development to staging/production
+ssh 1.ent-1ab23cd4efghi-prod-a1bb2cd@ssh.eu-3.platform.sh 'mysqldump --single-transaction -u user -h database.internal main | gzip' > database.gz
```

Comment:
:lady_beetle: 
Note the `database.internal`. That's the `host`. That needs to be in the list above. Make sure it's present anywhere it needs to be.